gallery/src/components/NxGlobalSidebarFooter/NxGlobalSidebarFooterScrollingExample.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {
+  NxGlobalSidebar,
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxGlobalSidebarFooter,
+  useToggle,
+  NxP,
+  NxH3,
+  NxPageMain,
+  NxTreeView,
+  NxCheckbox,
+  NxRadio,
+  NxTreeViewChild }
+  from '@sonatype/react-shared-components';
+import { faArrowLeft, faArrowRight, faLink, faExternalLinkSquareAlt } from '@fortawesome/free-solid-svg-icons';
+
+const logoImg = require('../../assets/images/logo-plaid-villain-text.png');
+
+export default function NxGlobalSidebarFooterScrollingExample() {
```

Comment:
Are there meaningful differences _in the footer_ between this example and the basic one?  If not this feels redundant/misnamed.  We already demonstrate the middle section scrolling on the sidebar page

Reply:
> Are there meaningful differences _in the footer_ between this example and the basic one? If not this feels redundant/misnamed. We already demonstrate the middle section scrolling on the sidebar page

https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/ce9878ac59ecb6215142c3d38f17850a334d360d